### PR TITLE
Gate other expenses questions

### PR DESCRIPTION
**Pull request by cehsu**
Connects to part two of department-of-veterans-affairs/vets.gov-team#3646
__________________________________________________________________________________________________________
### Veteran view
screen shot 2017-07-05 at 4 39 49 pm

__________________________________________________________________________________________________________

### Spouse view
screen shot 2017-07-05 at 4 40 17 pm

__________________________________________________________________________________________________________

### Dependent/child view
screen shot 2017-07-05 at 4 40 53 pm

__________________________________________________________________________________________________________

### Expanded view
screen shot 2017-07-03 at 11 13 26 am

### Warning
screen shot 2017-07-05 at 4 14 54 pm

### CODE REVIEW

File: src/js/pensions/config/form.js [hunk 1]
```diff
@@ -1262,7 +1287,15 @@ const formConfig = {
             dependents: {
               items: {
                 'ui:title': createHouseholdMemberTitle('fullName', 'Medical, legal, or other unreimbursed expenses'),
-                otherExpenses: otherExpensesUI
+                'view:hasOtherExpenses': {
+                  'ui:title': 'Does your child have any medical, legal or other unreimbursed expenses? (You will need to supply receipts)',
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Should this be changed to 'dependents'? Or is 'child' ok here?

File: src/js/pensions/config/form.js [hunk 2]
```diff
@@ -1140,12 +1151,23 @@ const formConfig = {
           schema: {
             type: 'object',
             properties: {
+              'view:hasOtherExpenses': {
```
Comment (jbalboni) [inline src/js/pensions/config/form.js]: We normally make these gating questions required.

File: src/js/pensions/config/form.js (hunk 1 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: I think we say child everywhere else, so child seems fine.

**Review by jbalboni — APPROVED** [review]
I think the `hasOtherExpenses` questions should be required, otherwise looks good.

File: src/js/pensions/config/form.js (hunk 2 above)
Comment (cehsu) [inline src/js/pensions/config/form.js]: Thanks @jbalboni, change made.

File: src/js/pensions/config/form.js [hunk 3]
```diff
@@ -1070,13 +1070,25 @@ const formConfig = {
...
+                type: 'boolean'
+              },
               otherExpenses
             }
           },
           uiSchema: {
             'ui:title': createHouseholdMemberTitle('veteranFullName', 'Medical, legal, or other unreimbursed expenses'),
-            otherExpenses: otherExpensesUI
+            'view:hasOtherExpenses': {
+              'ui:title': 'Do you have any medical, legal or other unreimbursed expenses? (You will need to supply receipts)',
```
Comment (cvalarida) [inline src/js/pensions/config/form.js]: The grammarian in me wants a period after "receipts." The programmer in me hates that the period in the previous sentence goes inside the quotes.

**Review by cvalarida — APPROVED** [review]
Looks good to me. I'd like to verify with @goldenmeanie that the text is right. In the original ticket, it had the name inserted in the text, but since this is right underneath the name in the header....I'm thinking it's alright..? What say you, @goldenmeanie?

File: src/js/pensions/config/form.js (hunk 3 above)
Comment (cehsu) [inline src/js/pensions/config/form.js]: Now you've got me wondering whether this belongs in a warning like burial transportation receipts. @goldenmeanie would that make sense here?

Comment (goldenmeanie) [inline src/js/pensions/config/form.js]: Yes @cehsu that makes sense to me.

**Pull request merged**